pybamm/models/submodels/active_material/varying_active_material_full.py

```diff
@@ -19,6 +19,15 @@ class VaryingFull(BaseModel):
         Additional options to pass to the model
 
     **Extends:** :class:`pybamm.active_material.BaseModel`
+
+    References
+    ----------
+    .. [1] Ai, W., Kraft, L., Sturm, J., Jossen, A., & Wu, B. (2019). Electrochemical
+           Thermal-Mechanical Modelling of Stress Inhomogeneity in Lithium-Ion Pouch
+           Cells. Journal of The Electrochemical Society, 167(1), 013512.
+    .. [2] Reniers, J. M., Mulder, G., & Howey, D. A. (2019). Review and performance
+           comparison of mechanical-chemical degradation models for lithium-ion
+           batteries. Journal of The Electrochemical Society, 166(14), A3189.
```

Comment:
Maybe this submodel should be called something like "StressDriven"?
Unsure. It depends whether, if there is another mechanism for LAM, we want to add it on top of this stress-driven mechanism (in which case there is a single submodel for LAM which adds up all the contributions to LAM), or instead of (in which case there is a different submodel for each LAM mechanism)

Reply:
There are indeed other mechanisms of LAM in literature, e.g. cathode dissolution, but they are not being implemented. It is not bad to leave options for them anyway. How does it sound if I put the submodel to a function with an output of `j_LAM_stress`?